<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MABL:: Real Time Serial Plotter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,2,2,0,0,10,0,0">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBoxADCRes"/>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="pushButtonStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Baud Rate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditPortName">
        <property name="text">
         <string>/dev/ttyACM0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="checkBoxSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="comboBoxBaudRate"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ADC Resolution (Y axis)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEditInterval"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Timer Interval (sec)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string># Channels</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QComboBox" name="comboBoxChannel"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
